python/ray/tune/utils/file_transfer.py

```diff
@@ -217,10 +233,15 @@ def delete_on_node(
         Boolean indicating if deletion succeeded, or Ray future
         for scheduled delete task.
     """
+
+    node_id = _get_node_id_from_node_ip(node_ip)
+
     delete_task = _remote_delete_path.options(
         num_cpus=0,
-        resources={f"node:{node_ip}": 0.01},
-        scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group=None),
+        scheduling_strategy=NodeAffinitySchedulingStrategy(
+            node_id=node_id,
+            soft=False,
+        ),
```

Comment:
Should we also have a utility function for `force_on_node_ip/id`?

Reply:
Makes sense! I added a `_force_on_node` that the caller can use in combination with `_get_node_id_from_node_ip`.